[BUG] rapids_it-matrix-dev-github delta_lake_time_travel_test.py test failures DATAGEN_SEED=1751797993

**Describe the bug**
rapids_it-matrix-dev-github/280 tests failing:

delta_lake_time_travel_test.py::test_time_travel_on_non_existing_table
delta_lake_time_travel_test.py::test_time_travel_sql_timestamp[in_commit_ts=False]
delta_lake_time_travel_test.py::test_time_travel_sql_version[in_commit_ts=False]

DATAGEN_SEED=1751797993

Details
```
[2025-07-06T10:59:04.204Z] =========================== short test summary info ============================

[2025-07-06T10:59:04.204Z] FAILED ../../src/main/python/delta_lake_time_travel_test.py::test_time_travel_on_non_existing_table[DATAGEN_SEED=1751797993, TZ=UTC] - AssertionError: Expected error 'AnalysisException' did not appear in 'pyspark.sql.utils.ParseException: 

[2025-07-06T10:59:04.204Z] mismatched input 'AS' expecting {<EOF>, ';'}(line 1, pos 35)

[2025-07-06T10:59:04.204Z] 

[2025-07-06T10:59:04.204Z] == SQL ==

[2025-07-06T10:59:04.204Z] SELECT * FROM not_existing VERSION AS OF 0

[2025-07-06T10:59:04.204Z] -----------------------------------^^^'

[2025-07-06T10:59:04.204Z] FAILED ../../src/main/python/delta_lake_time_travel_test.py::test_time_travel_sql_version[in_commit_ts=False][DATAGEN_SEED=1751797993, TZ=UTC, IGNORE_ORDER({'local': True}), ALLOW_NON_GPU(DeserializeToObjectExec,ShuffleExchangeExec,FileSourceScanExec,FilterExec,MapPartitionsExec,MapElementsExec,ObjectHashAggregateExec,ProjectExec,SerializeFromObjectExec,SortExec)] - pyspark.sql.utils.ParseException: 

[2025-07-06T10:59:04.204Z] mismatched input 'AS' expecting {<EOF>, ';'}(line 1, pos 48)

[2025-07-06T10:59:04.204Z] 

[2025-07-06T10:59:04.204Z] == SQL ==

[2025-07-06T10:59:04.204Z] SELECT * FROM tmp_table_gw0_455688323_0 VERSION AS OF 0

[2025-07-06T10:59:04.204Z] ------------------------------------------------^^^

[2025-07-06T10:59:04.204Z] FAILED ../../src/main/python/delta_lake_time_travel_test.py::test_time_travel_sql_timestamp[in_commit_ts=False][DATAGEN_SEED=1751797993, TZ=UTC, IGNORE_ORDER({'local': True}), ALLOW_NON_GPU(DeserializeToObjectExec,ShuffleExchangeExec,FileSourceScanExec,FilterExec,MapPartitionsExec,MapElementsExec,ObjectHashAggregateExec,ProjectExec,SerializeFromObjectExec,SortExec)] - pyspark.sql.utils.ParseException: 

[2025-07-06T10:59:04.204Z] mismatched input 'AS' expecting {<EOF>, ';'}(line 1, pos 51)

[2025-07-06T10:59:04.204Z] 

[2025-07-06T10:59:04.204Z] == SQL ==

[2025-07-06T10:59:04.204Z] SELECT * FROM tmp_table_gw0_1867957714_0 TIMESTAMP AS OF '2025-07-06T10:49:09.730000'

[2025-07-06T10:59:04.204Z] ---------------------------------------------------^^^

```

```
[2025-07-06T10:52:41.478Z] + run_delta_lake_tests
[2025-07-06T10:52:41.478Z] + echo 'run_delta_lake_tests SPARK_VER = 3.4.1'
[2025-07-06T10:52:41.478Z] run_delta_lake_tests SPARK_VER = 3.4.1
[2025-07-06T10:52:41.478Z] + SPARK_32X_PATTERN='(3\.2\.[0-9])'
[2025-07-06T10:52:41.478Z] + SPARK_33X_PATTERN='(3\.3\.[0-9])'
[2025-07-06T10:52:41.478Z] + SPARK_34X_PATTERN='(3\.4\.[0-9])'
[2025-07-06T10:52:41.478Z] + SPARK_35X_PATTERN='(3\.5\.[3-9])'
[2025-07-06T10:52:41.478Z] + [[ 3.4.1 =~ (3\.2\.[0-9]) ]]
[2025-07-06T10:52:41.478Z] + [[ 3.4.1 =~ (3\.3\.[0-9]) ]]
[2025-07-06T10:52:41.478Z] + [[ 3.4.1 =~ (3\.4\.[0-9]) ]]
[2025-07-06T10:52:41.478Z] + DELTA_LAKE_VERSIONS=2.4.0
[2025-07-06T10:52:41.478Z] + [[ 3.4.1 =~ (3\.5\.[3-9]) ]]
[2025-07-06T10:52:41.478Z] + '[' -z 2.4.0 ']'
[2025-07-06T10:52:41.478Z] + for v in $DELTA_LAKE_VERSIONS
[2025-07-06T10:52:41.478Z] + echo 'Running Delta Lake tests for Delta Lake version 2.4.0'
[2025-07-06T10:52:41.478Z] Running Delta Lake tests for Delta Lake version 2.4.0
[2025-07-06T10:52:41.478Z] + [[ 2.4.0 == \3\.\3\.\0 ]]
[2025-07-06T10:52:41.478Z] + DELTA_JAR=io.delta:delta-core_2.12:2.4.0
[2025-07-06T10:52:41.478Z] + PYSP_TEST_spark_jars_packages=io.delta:delta-core_2.12:2.4.0
[2025-07-06T10:52:41.478Z] + PYSP_TEST_spark_sql_extensions=io.delta.sql.DeltaSparkSessionExtension
[2025-07-06T10:52:41.478Z] + PYSP_TEST_spark_sql_catalog_spark__catalog=org.apache.spark.sql.delta.catalog.DeltaCatalog
[2025-07-06T10:52:41.478Z] + ./run_pyspark_from_build.sh -m delta_lake --delta_lake
```

--- a/integration_tests/src/main/python/delta_lake_time_travel_test.py
+++ b/integration_tests/src/main/python/delta_lake_time_travel_test.py
@@ -19,7 +19,7 @@ from asserts import *
 from data_gen import *
 from delta_lake_utils import delta_meta_allow
 from marks import *
-from spark_session import with_gpu_session, is_spark_353_or_later
+from spark_session import with_gpu_session, is_spark_353_or_later, is_before_spark_330
 from dataclasses import dataclass
 
 
@@ -126,6 +126,7 @@ def test_time_travel_df_version(spark_tmp_path, spark_tmp_table_factory, in_comm
 @allow_non_gpu(*delta_meta_allow)
 @delta_lake
 @ignore_order(local=True)
+@pytest.mark.skipif(is_before_spark_330(), "AS OF sql syntax is only supported after spark 3.3.0")
 @pytest.mark.parametrize("in_commit_ts", enable_in_commit_ts(), ids=in_commit_ts_param_id)
 def test_time_travel_sql_version(spark_tmp_path, spark_tmp_table_factory, in_commit_ts):
     result = do_set_up_tables_for_time_travel(spark_tmp_path, spark_tmp_table_factory,
@@ -159,6 +160,7 @@ def test_time_travel_df_timestamp(spark_tmp_path, spark_tmp_table_factory, in_co
 @allow_non_gpu(*delta_meta_allow)
 @delta_lake
 @ignore_order(local=True)
+@pytest.mark.skipif(is_before_spark_330(), "AS OF sql syntax is only supported after spark 3.3.0")
 @pytest.mark.parametrize("in_commit_ts", enable_in_commit_ts(), ids=in_commit_ts_param_id)
 def test_time_travel_sql_timestamp(spark_tmp_path, spark_tmp_table_factory, in_commit_ts):
     result = do_set_up_tables_for_time_travel(spark_tmp_path, spark_tmp_table_factory,

--- a/integration_tests/src/main/python/delta_lake_time_travel_test.py
+++ b/integration_tests/src/main/python/delta_lake_time_travel_test.py
@@ -126,7 +126,7 @@ def test_time_travel_df_version(spark_tmp_path, spark_tmp_table_factory, in_comm
 @allow_non_gpu(*delta_meta_allow)
 @delta_lake
 @ignore_order(local=True)
-@pytest.mark.skipif(is_before_spark_330(), "AS OF sql syntax is only supported after spark 3.3.0")
+@pytest.mark.skipif(is_before_spark_330(), reason = "AS OF sql syntax is only supported after spark.3.0")
 @pytest.mark.parametrize("in_commit_ts", enable_in_commit_ts(), ids=in_commit_ts_param_id)
 def test_time_travel_sql_version(spark_tmp_path, spark_tmp_table_factory, in_commit_ts):
     result = do_set_up_tables_for_time_travel(spark_tmp_path, spark_tmp_table_factory,
@@ -160,7 +160,7 @@ def test_time_travel_df_timestamp(spark_tmp_path, spark_tmp_table_factory, in_co
 @allow_non_gpu(*delta_meta_allow)
 @delta_lake
 @ignore_order(local=True)
-@pytest.mark.skipif(is_before_spark_330(), "AS OF sql syntax is only supported after spark 3.3.0")
+@pytest.mark.skipif(is_before_spark_330(), reason = "AS OF sql syntax is only supported after spark.3.0")
 @pytest.mark.parametrize("in_commit_ts", enable_in_commit_ts(), ids=in_commit_ts_param_id)
 def test_time_travel_sql_timestamp(spark_tmp_path, spark_tmp_table_factory, in_commit_ts):
     result = do_set_up_tables_for_time_travel(spark_tmp_path, spark_tmp_table_factory,

